### PR TITLE
Emits event standingSellOrder with packedOrder data 

### PR DESCRIPTION
**Pull request by josojo**
Emits event standingSellOrder with packedOrder data as originally requested by Felix

testplan:
unit tests

### CODE REVIEW

**Review by fleupold — APPROVED** [review]
👏 

Thanks, this should unblock the listener PR.

File: test/snapp_auction.js [hunk 1]
```diff
@@ -524,12 +544,10 @@ contract("SnappAuction", async (accounts) => {
       const order = encodeOrder(0, 1, 2, 3)
       const tx = await instance.placeSellOrders(order, { from: user_1 })
       const eventLog = tx.logs
-
-      const buyToken = eventLog[0].args.buyToken
       const sellToken = eventLog[0].args.sellToken
+      const buyToken = eventLog[0].args.buyToken
```
Comment (bh2smith) [inline test/snapp_auction.js]: I don't see why this was switched! Would prefer if it was switched back.

File: test/snapp_auction.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const {
 
 const {
   isActive,
-  encodeOrder }  = require("./snapp_utils.js")
+  encodeOrder}  = require("./snapp_utils.js")
```
Comment (bh2smith) [inline test/snapp_auction.js]:
```suggestion
  encodeOrder }  = require("./snapp_utils.js")
```

**Review by bh2smith — APPROVED** [review]
Looks good, I only had a few tiny comments about some unnecessary changes that don't really contribute to the point of the PR. Very cool!

**Pull request merged**